target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst

```diff
@@ -0,0 +1 @@
+com\murabi\api\MurabiServlet.class
```

Comment:
[major] Don't commit anything from the target dir since it contains only items that are built by maven.